tests/utils/aws-cleanup.js

```diff
@@ -0,0 +1,78 @@
+// NOTE: This script requires Node.js > 8 to run since it uses
+// modern Node.js / JavaScript features such as async / await
+
+const {
+  logger,
+  testServiceIdentifier,
+  findStacks,
+  listStackResources,
+  findRestApis,
+  deleteBucket,
+  deleteStack,
+  deleteRestApi,
+} = require('./index');
+
+async function findDeploymentBuckets(stacks) {
+  const buckets = [];
+  for (const stack of stacks) {
```

Comment:
Why not handle each stack in parallel? I take it'll be significantly faster.

Reply:
Let's try that :+1: Not sure if we'll hit some unrecoverable AWS limits though (had some issues with API removals, hence the sequential removal).

Reply:
In that case AWS will fail with error that has `retryable: true`. So we may eventually use some wrapper that retries on such occurrence (we already do something like that when deploying services)